### PR TITLE
created invoices article

### PR DESCRIPTION
**Pull request by BoVice**

### CODE REVIEW

**Comment by moritzplassnig** [comment]
Content = *thumbsup*

@bowkerke Any comments?

**Comment by bowkerke** [comment]
Looks good to me, I have one question, is there a reason we have a line break after the word "and"?

File: _posts/administration/2016-03-17-invoices.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - **Personal Account**: Select _Account Settings_
+
+    - **Organization Account**: Select _Organization Name_
+
+- Select _Invoice_ link 
+
+Here, you will see the history of all previous invoices. All invoices can be downloaded as a PDF file.
+
+### Receive invoices in another email address
+You can also change the email address to receive the invoices in **Manage Subscriptions** > _Update credit card and billing address_.
```
Comment (BoVice) [inline _posts/administration/2016-03-17-invoices.md]: @bowkerke  Are you referencing line 32? it's on the same line

**Comment by bowkerke** [comment]
It was from an old screen shot, you can ignore my comment. This looks good.

**Pull request merged**